tests/v2/AuthenticatedKernelTestCase.php

```diff
@@ -21,4 +27,16 @@ protected function loginUser(string $email): void
         $session->set('_security_main', serialize($token));
         $session->save();
     }
+
+    protected function getTestUserFromDb(string $email): User
+    {
+        return UserFactory::find(['email' => $email])->object();
+    }
+
+    protected function getPrivateMethod(string $method): \ReflectionMethod
+    {
+        $reflector = new \ReflectionClass(DocumentManager::class);
```

Comment:
I don't think this is in the right place (or maybe you should pass the class as argument?)

Reply:
You're right it's a mistake